2024-04-24 16:45:58,036 ERROR scheduler Exception in Enqueue Events for Site erpnext14.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 94, in enqueue_events
    queued_jobs = get_jobs(site=site, key="job_type").get(site) or []
  File "apps/frappe/frappe/utils/background_jobs.py", line 298, in get_jobs
    jobs = q.jobs + get_running_jobs_in_queue(q)
  File "apps/frappe/frappe/utils/background_jobs.py", line 335, in get_running_jobs_in_queue
    workers = get_workers(queue)
  File "apps/frappe/frappe/utils/background_jobs.py", line 327, in get_workers
    return Worker.all(queue=queue)
  File "env/lib/python3.10/site-packages/rq/worker.py", line 257, in all
    workers = [
  File "env/lib/python3.10/site-packages/rq/worker.py", line 258, in <listcomp>
    cls.find_by_key(
  File "env/lib/python3.10/site-packages/rq/worker.py", line 685, in find_by_key
    worker.refresh()
  File "env/lib/python3.10/site-packages/rq/worker.py", line 1041, in refresh
    queues = as_text(queues)
  File "env/lib/python3.10/site-packages/rq/utils.py", line 58, in as_text
    raise ValueError('Unknown type %r' % type(v))
ValueError: Unknown type <class 'NoneType'>

2024-04-24 16:46:58,193 ERROR scheduler Exception in Enqueue Events for Site erpnext14.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 94, in enqueue_events
    queued_jobs = get_jobs(site=site, key="job_type").get(site) or []
  File "apps/frappe/frappe/utils/background_jobs.py", line 298, in get_jobs
    jobs = q.jobs + get_running_jobs_in_queue(q)
  File "apps/frappe/frappe/utils/background_jobs.py", line 335, in get_running_jobs_in_queue
    workers = get_workers(queue)
  File "apps/frappe/frappe/utils/background_jobs.py", line 327, in get_workers
    return Worker.all(queue=queue)
  File "env/lib/python3.10/site-packages/rq/worker.py", line 257, in all
    workers = [
  File "env/lib/python3.10/site-packages/rq/worker.py", line 258, in <listcomp>
    cls.find_by_key(
  File "env/lib/python3.10/site-packages/rq/worker.py", line 685, in find_by_key
    worker.refresh()
  File "env/lib/python3.10/site-packages/rq/worker.py", line 1041, in refresh
    queues = as_text(queues)
  File "env/lib/python3.10/site-packages/rq/utils.py", line 58, in as_text
    raise ValueError('Unknown type %r' % type(v))
ValueError: Unknown type <class 'NoneType'>

2024-04-24 16:47:58,308 ERROR scheduler Exception in Enqueue Events for Site erpnext14.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 94, in enqueue_events
    queued_jobs = get_jobs(site=site, key="job_type").get(site) or []
  File "apps/frappe/frappe/utils/background_jobs.py", line 298, in get_jobs
    jobs = q.jobs + get_running_jobs_in_queue(q)
  File "apps/frappe/frappe/utils/background_jobs.py", line 335, in get_running_jobs_in_queue
    workers = get_workers(queue)
  File "apps/frappe/frappe/utils/background_jobs.py", line 327, in get_workers
    return Worker.all(queue=queue)
  File "env/lib/python3.10/site-packages/rq/worker.py", line 257, in all
    workers = [
  File "env/lib/python3.10/site-packages/rq/worker.py", line 258, in <listcomp>
    cls.find_by_key(
  File "env/lib/python3.10/site-packages/rq/worker.py", line 685, in find_by_key
    worker.refresh()
  File "env/lib/python3.10/site-packages/rq/worker.py", line 1041, in refresh
    queues = as_text(queues)
  File "env/lib/python3.10/site-packages/rq/utils.py", line 58, in as_text
    raise ValueError('Unknown type %r' % type(v))
ValueError: Unknown type <class 'NoneType'>

2024-04-24 16:48:58,425 ERROR scheduler Exception in Enqueue Events for Site erpnext14.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 94, in enqueue_events
    queued_jobs = get_jobs(site=site, key="job_type").get(site) or []
  File "apps/frappe/frappe/utils/background_jobs.py", line 298, in get_jobs
    jobs = q.jobs + get_running_jobs_in_queue(q)
  File "apps/frappe/frappe/utils/background_jobs.py", line 335, in get_running_jobs_in_queue
    workers = get_workers(queue)
  File "apps/frappe/frappe/utils/background_jobs.py", line 327, in get_workers
    return Worker.all(queue=queue)
  File "env/lib/python3.10/site-packages/rq/worker.py", line 257, in all
    workers = [
  File "env/lib/python3.10/site-packages/rq/worker.py", line 258, in <listcomp>
    cls.find_by_key(
  File "env/lib/python3.10/site-packages/rq/worker.py", line 685, in find_by_key
    worker.refresh()
  File "env/lib/python3.10/site-packages/rq/worker.py", line 1041, in refresh
    queues = as_text(queues)
  File "env/lib/python3.10/site-packages/rq/utils.py", line 58, in as_text
    raise ValueError('Unknown type %r' % type(v))
ValueError: Unknown type <class 'NoneType'>

2024-04-24 16:49:58,466 ERROR scheduler Exception in Enqueue Events for Site erpnext14.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 94, in enqueue_events
    queued_jobs = get_jobs(site=site, key="job_type").get(site) or []
  File "apps/frappe/frappe/utils/background_jobs.py", line 298, in get_jobs
    jobs = q.jobs + get_running_jobs_in_queue(q)
  File "apps/frappe/frappe/utils/background_jobs.py", line 335, in get_running_jobs_in_queue
    workers = get_workers(queue)
  File "apps/frappe/frappe/utils/background_jobs.py", line 327, in get_workers
    return Worker.all(queue=queue)
  File "env/lib/python3.10/site-packages/rq/worker.py", line 257, in all
    workers = [
  File "env/lib/python3.10/site-packages/rq/worker.py", line 258, in <listcomp>
    cls.find_by_key(
  File "env/lib/python3.10/site-packages/rq/worker.py", line 685, in find_by_key
    worker.refresh()
  File "env/lib/python3.10/site-packages/rq/worker.py", line 1041, in refresh
    queues = as_text(queues)
  File "env/lib/python3.10/site-packages/rq/utils.py", line 58, in as_text
    raise ValueError('Unknown type %r' % type(v))
ValueError: Unknown type <class 'NoneType'>

2024-04-24 16:50:58,533 ERROR scheduler Exception in Enqueue Events for Site erpnext14.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 94, in enqueue_events
    queued_jobs = get_jobs(site=site, key="job_type").get(site) or []
  File "apps/frappe/frappe/utils/background_jobs.py", line 298, in get_jobs
    jobs = q.jobs + get_running_jobs_in_queue(q)
  File "apps/frappe/frappe/utils/background_jobs.py", line 335, in get_running_jobs_in_queue
    workers = get_workers(queue)
  File "apps/frappe/frappe/utils/background_jobs.py", line 327, in get_workers
    return Worker.all(queue=queue)
  File "env/lib/python3.10/site-packages/rq/worker.py", line 257, in all
    workers = [
  File "env/lib/python3.10/site-packages/rq/worker.py", line 258, in <listcomp>
    cls.find_by_key(
  File "env/lib/python3.10/site-packages/rq/worker.py", line 685, in find_by_key
    worker.refresh()
  File "env/lib/python3.10/site-packages/rq/worker.py", line 1041, in refresh
    queues = as_text(queues)
  File "env/lib/python3.10/site-packages/rq/utils.py", line 58, in as_text
    raise ValueError('Unknown type %r' % type(v))
ValueError: Unknown type <class 'NoneType'>

2024-04-24 16:51:58,652 ERROR scheduler Exception in Enqueue Events for Site erpnext14.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 94, in enqueue_events
    queued_jobs = get_jobs(site=site, key="job_type").get(site) or []
  File "apps/frappe/frappe/utils/background_jobs.py", line 298, in get_jobs
    jobs = q.jobs + get_running_jobs_in_queue(q)
  File "apps/frappe/frappe/utils/background_jobs.py", line 335, in get_running_jobs_in_queue
    workers = get_workers(queue)
  File "apps/frappe/frappe/utils/background_jobs.py", line 327, in get_workers
    return Worker.all(queue=queue)
  File "env/lib/python3.10/site-packages/rq/worker.py", line 257, in all
    workers = [
  File "env/lib/python3.10/site-packages/rq/worker.py", line 258, in <listcomp>
    cls.find_by_key(
  File "env/lib/python3.10/site-packages/rq/worker.py", line 685, in find_by_key
    worker.refresh()
  File "env/lib/python3.10/site-packages/rq/worker.py", line 1041, in refresh
    queues = as_text(queues)
  File "env/lib/python3.10/site-packages/rq/utils.py", line 58, in as_text
    raise ValueError('Unknown type %r' % type(v))
ValueError: Unknown type <class 'NoneType'>

2024-04-24 16:52:58,775 ERROR scheduler Exception in Enqueue Events for Site erpnext14.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 94, in enqueue_events
    queued_jobs = get_jobs(site=site, key="job_type").get(site) or []
  File "apps/frappe/frappe/utils/background_jobs.py", line 298, in get_jobs
    jobs = q.jobs + get_running_jobs_in_queue(q)
  File "apps/frappe/frappe/utils/background_jobs.py", line 335, in get_running_jobs_in_queue
    workers = get_workers(queue)
  File "apps/frappe/frappe/utils/background_jobs.py", line 327, in get_workers
    return Worker.all(queue=queue)
  File "env/lib/python3.10/site-packages/rq/worker.py", line 257, in all
    workers = [
  File "env/lib/python3.10/site-packages/rq/worker.py", line 258, in <listcomp>
    cls.find_by_key(
  File "env/lib/python3.10/site-packages/rq/worker.py", line 685, in find_by_key
    worker.refresh()
  File "env/lib/python3.10/site-packages/rq/worker.py", line 1041, in refresh
    queues = as_text(queues)
  File "env/lib/python3.10/site-packages/rq/utils.py", line 58, in as_text
    raise ValueError('Unknown type %r' % type(v))
ValueError: Unknown type <class 'NoneType'>

2024-04-24 16:53:58,800 ERROR scheduler Exception in Enqueue Events for Site erpnext14.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 94, in enqueue_events
    queued_jobs = get_jobs(site=site, key="job_type").get(site) or []
  File "apps/frappe/frappe/utils/background_jobs.py", line 298, in get_jobs
    jobs = q.jobs + get_running_jobs_in_queue(q)
  File "apps/frappe/frappe/utils/background_jobs.py", line 335, in get_running_jobs_in_queue
    workers = get_workers(queue)
  File "apps/frappe/frappe/utils/background_jobs.py", line 327, in get_workers
    return Worker.all(queue=queue)
  File "env/lib/python3.10/site-packages/rq/worker.py", line 257, in all
    workers = [
  File "env/lib/python3.10/site-packages/rq/worker.py", line 258, in <listcomp>
    cls.find_by_key(
  File "env/lib/python3.10/site-packages/rq/worker.py", line 685, in find_by_key
    worker.refresh()
  File "env/lib/python3.10/site-packages/rq/worker.py", line 1041, in refresh
    queues = as_text(queues)
  File "env/lib/python3.10/site-packages/rq/utils.py", line 58, in as_text
    raise ValueError('Unknown type %r' % type(v))
ValueError: Unknown type <class 'NoneType'>

2024-04-24 16:54:58,863 ERROR scheduler Exception in Enqueue Events for Site erpnext14.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 94, in enqueue_events
    queued_jobs = get_jobs(site=site, key="job_type").get(site) or []
  File "apps/frappe/frappe/utils/background_jobs.py", line 298, in get_jobs
    jobs = q.jobs + get_running_jobs_in_queue(q)
  File "apps/frappe/frappe/utils/background_jobs.py", line 335, in get_running_jobs_in_queue
    workers = get_workers(queue)
  File "apps/frappe/frappe/utils/background_jobs.py", line 327, in get_workers
    return Worker.all(queue=queue)
  File "env/lib/python3.10/site-packages/rq/worker.py", line 257, in all
    workers = [
  File "env/lib/python3.10/site-packages/rq/worker.py", line 258, in <listcomp>
    cls.find_by_key(
  File "env/lib/python3.10/site-packages/rq/worker.py", line 685, in find_by_key
    worker.refresh()
  File "env/lib/python3.10/site-packages/rq/worker.py", line 1041, in refresh
    queues = as_text(queues)
  File "env/lib/python3.10/site-packages/rq/utils.py", line 58, in as_text
    raise ValueError('Unknown type %r' % type(v))
ValueError: Unknown type <class 'NoneType'>

2024-04-24 16:55:58,929 ERROR scheduler Exception in Enqueue Events for Site erpnext14.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 94, in enqueue_events
    queued_jobs = get_jobs(site=site, key="job_type").get(site) or []
  File "apps/frappe/frappe/utils/background_jobs.py", line 298, in get_jobs
    jobs = q.jobs + get_running_jobs_in_queue(q)
  File "apps/frappe/frappe/utils/background_jobs.py", line 335, in get_running_jobs_in_queue
    workers = get_workers(queue)
  File "apps/frappe/frappe/utils/background_jobs.py", line 327, in get_workers
    return Worker.all(queue=queue)
  File "env/lib/python3.10/site-packages/rq/worker.py", line 257, in all
    workers = [
  File "env/lib/python3.10/site-packages/rq/worker.py", line 258, in <listcomp>
    cls.find_by_key(
  File "env/lib/python3.10/site-packages/rq/worker.py", line 685, in find_by_key
    worker.refresh()
  File "env/lib/python3.10/site-packages/rq/worker.py", line 1041, in refresh
    queues = as_text(queues)
  File "env/lib/python3.10/site-packages/rq/utils.py", line 58, in as_text
    raise ValueError('Unknown type %r' % type(v))
ValueError: Unknown type <class 'NoneType'>

2024-04-24 16:56:58,949 ERROR scheduler Exception in Enqueue Events for Site erpnext14.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 94, in enqueue_events
    queued_jobs = get_jobs(site=site, key="job_type").get(site) or []
  File "apps/frappe/frappe/utils/background_jobs.py", line 298, in get_jobs
    jobs = q.jobs + get_running_jobs_in_queue(q)
  File "apps/frappe/frappe/utils/background_jobs.py", line 335, in get_running_jobs_in_queue
    workers = get_workers(queue)
  File "apps/frappe/frappe/utils/background_jobs.py", line 327, in get_workers
    return Worker.all(queue=queue)
  File "env/lib/python3.10/site-packages/rq/worker.py", line 257, in all
    workers = [
  File "env/lib/python3.10/site-packages/rq/worker.py", line 258, in <listcomp>
    cls.find_by_key(
  File "env/lib/python3.10/site-packages/rq/worker.py", line 685, in find_by_key
    worker.refresh()
  File "env/lib/python3.10/site-packages/rq/worker.py", line 1041, in refresh
    queues = as_text(queues)
  File "env/lib/python3.10/site-packages/rq/utils.py", line 58, in as_text
    raise ValueError('Unknown type %r' % type(v))
ValueError: Unknown type <class 'NoneType'>

2024-04-24 16:57:59,066 ERROR scheduler Exception in Enqueue Events for Site erpnext14.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 94, in enqueue_events
    queued_jobs = get_jobs(site=site, key="job_type").get(site) or []
  File "apps/frappe/frappe/utils/background_jobs.py", line 298, in get_jobs
    jobs = q.jobs + get_running_jobs_in_queue(q)
  File "apps/frappe/frappe/utils/background_jobs.py", line 335, in get_running_jobs_in_queue
    workers = get_workers(queue)
  File "apps/frappe/frappe/utils/background_jobs.py", line 327, in get_workers
    return Worker.all(queue=queue)
  File "env/lib/python3.10/site-packages/rq/worker.py", line 257, in all
    workers = [
  File "env/lib/python3.10/site-packages/rq/worker.py", line 258, in <listcomp>
    cls.find_by_key(
  File "env/lib/python3.10/site-packages/rq/worker.py", line 685, in find_by_key
    worker.refresh()
  File "env/lib/python3.10/site-packages/rq/worker.py", line 1041, in refresh
    queues = as_text(queues)
  File "env/lib/python3.10/site-packages/rq/utils.py", line 58, in as_text
    raise ValueError('Unknown type %r' % type(v))
ValueError: Unknown type <class 'NoneType'>

2024-04-24 16:58:59,185 ERROR scheduler Exception in Enqueue Events for Site erpnext14.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 94, in enqueue_events
    queued_jobs = get_jobs(site=site, key="job_type").get(site) or []
  File "apps/frappe/frappe/utils/background_jobs.py", line 298, in get_jobs
    jobs = q.jobs + get_running_jobs_in_queue(q)
  File "apps/frappe/frappe/utils/background_jobs.py", line 335, in get_running_jobs_in_queue
    workers = get_workers(queue)
  File "apps/frappe/frappe/utils/background_jobs.py", line 327, in get_workers
    return Worker.all(queue=queue)
  File "env/lib/python3.10/site-packages/rq/worker.py", line 257, in all
    workers = [
  File "env/lib/python3.10/site-packages/rq/worker.py", line 258, in <listcomp>
    cls.find_by_key(
  File "env/lib/python3.10/site-packages/rq/worker.py", line 685, in find_by_key
    worker.refresh()
  File "env/lib/python3.10/site-packages/rq/worker.py", line 1041, in refresh
    queues = as_text(queues)
  File "env/lib/python3.10/site-packages/rq/utils.py", line 58, in as_text
    raise ValueError('Unknown type %r' % type(v))
ValueError: Unknown type <class 'NoneType'>

2024-04-24 16:59:59,319 ERROR scheduler Exception in Enqueue Events for Site erpnext14.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 94, in enqueue_events
    queued_jobs = get_jobs(site=site, key="job_type").get(site) or []
  File "apps/frappe/frappe/utils/background_jobs.py", line 298, in get_jobs
    jobs = q.jobs + get_running_jobs_in_queue(q)
  File "apps/frappe/frappe/utils/background_jobs.py", line 335, in get_running_jobs_in_queue
    workers = get_workers(queue)
  File "apps/frappe/frappe/utils/background_jobs.py", line 327, in get_workers
    return Worker.all(queue=queue)
  File "env/lib/python3.10/site-packages/rq/worker.py", line 257, in all
    workers = [
  File "env/lib/python3.10/site-packages/rq/worker.py", line 258, in <listcomp>
    cls.find_by_key(
  File "env/lib/python3.10/site-packages/rq/worker.py", line 685, in find_by_key
    worker.refresh()
  File "env/lib/python3.10/site-packages/rq/worker.py", line 1041, in refresh
    queues = as_text(queues)
  File "env/lib/python3.10/site-packages/rq/utils.py", line 58, in as_text
    raise ValueError('Unknown type %r' % type(v))
ValueError: Unknown type <class 'NoneType'>

2024-04-24 17:00:59,387 ERROR scheduler Exception in Enqueue Events for Site erpnext14.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 94, in enqueue_events
    queued_jobs = get_jobs(site=site, key="job_type").get(site) or []
  File "apps/frappe/frappe/utils/background_jobs.py", line 298, in get_jobs
    jobs = q.jobs + get_running_jobs_in_queue(q)
  File "apps/frappe/frappe/utils/background_jobs.py", line 335, in get_running_jobs_in_queue
    workers = get_workers(queue)
  File "apps/frappe/frappe/utils/background_jobs.py", line 327, in get_workers
    return Worker.all(queue=queue)
  File "env/lib/python3.10/site-packages/rq/worker.py", line 257, in all
    workers = [
  File "env/lib/python3.10/site-packages/rq/worker.py", line 258, in <listcomp>
    cls.find_by_key(
  File "env/lib/python3.10/site-packages/rq/worker.py", line 685, in find_by_key
    worker.refresh()
  File "env/lib/python3.10/site-packages/rq/worker.py", line 1041, in refresh
    queues = as_text(queues)
  File "env/lib/python3.10/site-packages/rq/utils.py", line 58, in as_text
    raise ValueError('Unknown type %r' % type(v))
ValueError: Unknown type <class 'NoneType'>

2024-04-24 17:01:59,436 ERROR scheduler Exception in Enqueue Events for Site erpnext14.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 94, in enqueue_events
    queued_jobs = get_jobs(site=site, key="job_type").get(site) or []
  File "apps/frappe/frappe/utils/background_jobs.py", line 298, in get_jobs
    jobs = q.jobs + get_running_jobs_in_queue(q)
  File "apps/frappe/frappe/utils/background_jobs.py", line 335, in get_running_jobs_in_queue
    workers = get_workers(queue)
  File "apps/frappe/frappe/utils/background_jobs.py", line 327, in get_workers
    return Worker.all(queue=queue)
  File "env/lib/python3.10/site-packages/rq/worker.py", line 257, in all
    workers = [
  File "env/lib/python3.10/site-packages/rq/worker.py", line 258, in <listcomp>
    cls.find_by_key(
  File "env/lib/python3.10/site-packages/rq/worker.py", line 685, in find_by_key
    worker.refresh()
  File "env/lib/python3.10/site-packages/rq/worker.py", line 1041, in refresh
    queues = as_text(queues)
  File "env/lib/python3.10/site-packages/rq/utils.py", line 58, in as_text
    raise ValueError('Unknown type %r' % type(v))
ValueError: Unknown type <class 'NoneType'>

2024-04-24 17:02:59,479 ERROR scheduler Exception in Enqueue Events for Site erpnext14.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 94, in enqueue_events
    queued_jobs = get_jobs(site=site, key="job_type").get(site) or []
  File "apps/frappe/frappe/utils/background_jobs.py", line 298, in get_jobs
    jobs = q.jobs + get_running_jobs_in_queue(q)
  File "apps/frappe/frappe/utils/background_jobs.py", line 335, in get_running_jobs_in_queue
    workers = get_workers(queue)
  File "apps/frappe/frappe/utils/background_jobs.py", line 327, in get_workers
    return Worker.all(queue=queue)
  File "env/lib/python3.10/site-packages/rq/worker.py", line 257, in all
    workers = [
  File "env/lib/python3.10/site-packages/rq/worker.py", line 258, in <listcomp>
    cls.find_by_key(
  File "env/lib/python3.10/site-packages/rq/worker.py", line 685, in find_by_key
    worker.refresh()
  File "env/lib/python3.10/site-packages/rq/worker.py", line 1041, in refresh
    queues = as_text(queues)
  File "env/lib/python3.10/site-packages/rq/utils.py", line 58, in as_text
    raise ValueError('Unknown type %r' % type(v))
ValueError: Unknown type <class 'NoneType'>

2024-04-24 17:03:59,594 ERROR scheduler Exception in Enqueue Events for Site erpnext14.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 94, in enqueue_events
    queued_jobs = get_jobs(site=site, key="job_type").get(site) or []
  File "apps/frappe/frappe/utils/background_jobs.py", line 298, in get_jobs
    jobs = q.jobs + get_running_jobs_in_queue(q)
  File "apps/frappe/frappe/utils/background_jobs.py", line 335, in get_running_jobs_in_queue
    workers = get_workers(queue)
  File "apps/frappe/frappe/utils/background_jobs.py", line 327, in get_workers
    return Worker.all(queue=queue)
  File "env/lib/python3.10/site-packages/rq/worker.py", line 257, in all
    workers = [
  File "env/lib/python3.10/site-packages/rq/worker.py", line 258, in <listcomp>
    cls.find_by_key(
  File "env/lib/python3.10/site-packages/rq/worker.py", line 685, in find_by_key
    worker.refresh()
  File "env/lib/python3.10/site-packages/rq/worker.py", line 1041, in refresh
    queues = as_text(queues)
  File "env/lib/python3.10/site-packages/rq/utils.py", line 58, in as_text
    raise ValueError('Unknown type %r' % type(v))
ValueError: Unknown type <class 'NoneType'>

2024-04-24 17:04:59,710 ERROR scheduler Exception in Enqueue Events for Site erpnext14.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 94, in enqueue_events
    queued_jobs = get_jobs(site=site, key="job_type").get(site) or []
  File "apps/frappe/frappe/utils/background_jobs.py", line 298, in get_jobs
    jobs = q.jobs + get_running_jobs_in_queue(q)
  File "apps/frappe/frappe/utils/background_jobs.py", line 335, in get_running_jobs_in_queue
    workers = get_workers(queue)
  File "apps/frappe/frappe/utils/background_jobs.py", line 327, in get_workers
    return Worker.all(queue=queue)
  File "env/lib/python3.10/site-packages/rq/worker.py", line 257, in all
    workers = [
  File "env/lib/python3.10/site-packages/rq/worker.py", line 258, in <listcomp>
    cls.find_by_key(
  File "env/lib/python3.10/site-packages/rq/worker.py", line 685, in find_by_key
    worker.refresh()
  File "env/lib/python3.10/site-packages/rq/worker.py", line 1041, in refresh
    queues = as_text(queues)
  File "env/lib/python3.10/site-packages/rq/utils.py", line 58, in as_text
    raise ValueError('Unknown type %r' % type(v))
ValueError: Unknown type <class 'NoneType'>

2024-04-24 17:05:59,824 ERROR scheduler Exception in Enqueue Events for Site erpnext14.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 94, in enqueue_events
    queued_jobs = get_jobs(site=site, key="job_type").get(site) or []
  File "apps/frappe/frappe/utils/background_jobs.py", line 298, in get_jobs
    jobs = q.jobs + get_running_jobs_in_queue(q)
  File "apps/frappe/frappe/utils/background_jobs.py", line 335, in get_running_jobs_in_queue
    workers = get_workers(queue)
  File "apps/frappe/frappe/utils/background_jobs.py", line 327, in get_workers
    return Worker.all(queue=queue)
  File "env/lib/python3.10/site-packages/rq/worker.py", line 257, in all
    workers = [
  File "env/lib/python3.10/site-packages/rq/worker.py", line 258, in <listcomp>
    cls.find_by_key(
  File "env/lib/python3.10/site-packages/rq/worker.py", line 685, in find_by_key
    worker.refresh()
  File "env/lib/python3.10/site-packages/rq/worker.py", line 1041, in refresh
    queues = as_text(queues)
  File "env/lib/python3.10/site-packages/rq/utils.py", line 58, in as_text
    raise ValueError('Unknown type %r' % type(v))
ValueError: Unknown type <class 'NoneType'>

2024-04-24 17:06:59,859 ERROR scheduler Exception in Enqueue Events for Site erpnext14.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 94, in enqueue_events
    queued_jobs = get_jobs(site=site, key="job_type").get(site) or []
  File "apps/frappe/frappe/utils/background_jobs.py", line 298, in get_jobs
    jobs = q.jobs + get_running_jobs_in_queue(q)
  File "apps/frappe/frappe/utils/background_jobs.py", line 335, in get_running_jobs_in_queue
    workers = get_workers(queue)
  File "apps/frappe/frappe/utils/background_jobs.py", line 327, in get_workers
    return Worker.all(queue=queue)
  File "env/lib/python3.10/site-packages/rq/worker.py", line 257, in all
    workers = [
  File "env/lib/python3.10/site-packages/rq/worker.py", line 258, in <listcomp>
    cls.find_by_key(
  File "env/lib/python3.10/site-packages/rq/worker.py", line 685, in find_by_key
    worker.refresh()
  File "env/lib/python3.10/site-packages/rq/worker.py", line 1041, in refresh
    queues = as_text(queues)
  File "env/lib/python3.10/site-packages/rq/utils.py", line 58, in as_text
    raise ValueError('Unknown type %r' % type(v))
ValueError: Unknown type <class 'NoneType'>

2024-04-24 17:07:59,923 ERROR scheduler Exception in Enqueue Events for Site erpnext14.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 94, in enqueue_events
    queued_jobs = get_jobs(site=site, key="job_type").get(site) or []
  File "apps/frappe/frappe/utils/background_jobs.py", line 298, in get_jobs
    jobs = q.jobs + get_running_jobs_in_queue(q)
  File "apps/frappe/frappe/utils/background_jobs.py", line 335, in get_running_jobs_in_queue
    workers = get_workers(queue)
  File "apps/frappe/frappe/utils/background_jobs.py", line 327, in get_workers
    return Worker.all(queue=queue)
  File "env/lib/python3.10/site-packages/rq/worker.py", line 257, in all
    workers = [
  File "env/lib/python3.10/site-packages/rq/worker.py", line 258, in <listcomp>
    cls.find_by_key(
  File "env/lib/python3.10/site-packages/rq/worker.py", line 685, in find_by_key
    worker.refresh()
  File "env/lib/python3.10/site-packages/rq/worker.py", line 1041, in refresh
    queues = as_text(queues)
  File "env/lib/python3.10/site-packages/rq/utils.py", line 58, in as_text
    raise ValueError('Unknown type %r' % type(v))
ValueError: Unknown type <class 'NoneType'>

2024-04-24 17:08:59,949 ERROR scheduler Exception in Enqueue Events for Site erpnext14.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 94, in enqueue_events
    queued_jobs = get_jobs(site=site, key="job_type").get(site) or []
  File "apps/frappe/frappe/utils/background_jobs.py", line 298, in get_jobs
    jobs = q.jobs + get_running_jobs_in_queue(q)
  File "apps/frappe/frappe/utils/background_jobs.py", line 335, in get_running_jobs_in_queue
    workers = get_workers(queue)
  File "apps/frappe/frappe/utils/background_jobs.py", line 327, in get_workers
    return Worker.all(queue=queue)
  File "env/lib/python3.10/site-packages/rq/worker.py", line 257, in all
    workers = [
  File "env/lib/python3.10/site-packages/rq/worker.py", line 258, in <listcomp>
    cls.find_by_key(
  File "env/lib/python3.10/site-packages/rq/worker.py", line 685, in find_by_key
    worker.refresh()
  File "env/lib/python3.10/site-packages/rq/worker.py", line 1041, in refresh
    queues = as_text(queues)
  File "env/lib/python3.10/site-packages/rq/utils.py", line 58, in as_text
    raise ValueError('Unknown type %r' % type(v))
ValueError: Unknown type <class 'NoneType'>

2024-04-24 17:10:00,062 ERROR scheduler Exception in Enqueue Events for Site erpnext14.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 94, in enqueue_events
    queued_jobs = get_jobs(site=site, key="job_type").get(site) or []
  File "apps/frappe/frappe/utils/background_jobs.py", line 298, in get_jobs
    jobs = q.jobs + get_running_jobs_in_queue(q)
  File "apps/frappe/frappe/utils/background_jobs.py", line 335, in get_running_jobs_in_queue
    workers = get_workers(queue)
  File "apps/frappe/frappe/utils/background_jobs.py", line 327, in get_workers
    return Worker.all(queue=queue)
  File "env/lib/python3.10/site-packages/rq/worker.py", line 257, in all
    workers = [
  File "env/lib/python3.10/site-packages/rq/worker.py", line 258, in <listcomp>
    cls.find_by_key(
  File "env/lib/python3.10/site-packages/rq/worker.py", line 685, in find_by_key
    worker.refresh()
  File "env/lib/python3.10/site-packages/rq/worker.py", line 1041, in refresh
    queues = as_text(queues)
  File "env/lib/python3.10/site-packages/rq/utils.py", line 58, in as_text
    raise ValueError('Unknown type %r' % type(v))
ValueError: Unknown type <class 'NoneType'>

2024-04-24 17:11:00,180 ERROR scheduler Exception in Enqueue Events for Site erpnext14.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 94, in enqueue_events
    queued_jobs = get_jobs(site=site, key="job_type").get(site) or []
  File "apps/frappe/frappe/utils/background_jobs.py", line 298, in get_jobs
    jobs = q.jobs + get_running_jobs_in_queue(q)
  File "apps/frappe/frappe/utils/background_jobs.py", line 335, in get_running_jobs_in_queue
    workers = get_workers(queue)
  File "apps/frappe/frappe/utils/background_jobs.py", line 327, in get_workers
    return Worker.all(queue=queue)
  File "env/lib/python3.10/site-packages/rq/worker.py", line 257, in all
    workers = [
  File "env/lib/python3.10/site-packages/rq/worker.py", line 258, in <listcomp>
    cls.find_by_key(
  File "env/lib/python3.10/site-packages/rq/worker.py", line 685, in find_by_key
    worker.refresh()
  File "env/lib/python3.10/site-packages/rq/worker.py", line 1041, in refresh
    queues = as_text(queues)
  File "env/lib/python3.10/site-packages/rq/utils.py", line 58, in as_text
    raise ValueError('Unknown type %r' % type(v))
ValueError: Unknown type <class 'NoneType'>

2024-04-24 17:12:00,295 ERROR scheduler Exception in Enqueue Events for Site erpnext14.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 94, in enqueue_events
    queued_jobs = get_jobs(site=site, key="job_type").get(site) or []
  File "apps/frappe/frappe/utils/background_jobs.py", line 298, in get_jobs
    jobs = q.jobs + get_running_jobs_in_queue(q)
  File "apps/frappe/frappe/utils/background_jobs.py", line 335, in get_running_jobs_in_queue
    workers = get_workers(queue)
  File "apps/frappe/frappe/utils/background_jobs.py", line 327, in get_workers
    return Worker.all(queue=queue)
  File "env/lib/python3.10/site-packages/rq/worker.py", line 257, in all
    workers = [
  File "env/lib/python3.10/site-packages/rq/worker.py", line 258, in <listcomp>
    cls.find_by_key(
  File "env/lib/python3.10/site-packages/rq/worker.py", line 685, in find_by_key
    worker.refresh()
  File "env/lib/python3.10/site-packages/rq/worker.py", line 1041, in refresh
    queues = as_text(queues)
  File "env/lib/python3.10/site-packages/rq/utils.py", line 58, in as_text
    raise ValueError('Unknown type %r' % type(v))
ValueError: Unknown type <class 'NoneType'>

2024-04-24 17:13:00,340 ERROR scheduler Exception in Enqueue Events for Site erpnext14.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 94, in enqueue_events
    queued_jobs = get_jobs(site=site, key="job_type").get(site) or []
  File "apps/frappe/frappe/utils/background_jobs.py", line 298, in get_jobs
    jobs = q.jobs + get_running_jobs_in_queue(q)
  File "apps/frappe/frappe/utils/background_jobs.py", line 335, in get_running_jobs_in_queue
    workers = get_workers(queue)
  File "apps/frappe/frappe/utils/background_jobs.py", line 327, in get_workers
    return Worker.all(queue=queue)
  File "env/lib/python3.10/site-packages/rq/worker.py", line 257, in all
    workers = [
  File "env/lib/python3.10/site-packages/rq/worker.py", line 258, in <listcomp>
    cls.find_by_key(
  File "env/lib/python3.10/site-packages/rq/worker.py", line 685, in find_by_key
    worker.refresh()
  File "env/lib/python3.10/site-packages/rq/worker.py", line 1041, in refresh
    queues = as_text(queues)
  File "env/lib/python3.10/site-packages/rq/utils.py", line 58, in as_text
    raise ValueError('Unknown type %r' % type(v))
ValueError: Unknown type <class 'NoneType'>

2024-04-24 17:14:00,458 ERROR scheduler Exception in Enqueue Events for Site erpnext14.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 94, in enqueue_events
    queued_jobs = get_jobs(site=site, key="job_type").get(site) or []
  File "apps/frappe/frappe/utils/background_jobs.py", line 298, in get_jobs
    jobs = q.jobs + get_running_jobs_in_queue(q)
  File "apps/frappe/frappe/utils/background_jobs.py", line 335, in get_running_jobs_in_queue
    workers = get_workers(queue)
  File "apps/frappe/frappe/utils/background_jobs.py", line 327, in get_workers
    return Worker.all(queue=queue)
  File "env/lib/python3.10/site-packages/rq/worker.py", line 257, in all
    workers = [
  File "env/lib/python3.10/site-packages/rq/worker.py", line 258, in <listcomp>
    cls.find_by_key(
  File "env/lib/python3.10/site-packages/rq/worker.py", line 685, in find_by_key
    worker.refresh()
  File "env/lib/python3.10/site-packages/rq/worker.py", line 1041, in refresh
    queues = as_text(queues)
  File "env/lib/python3.10/site-packages/rq/utils.py", line 58, in as_text
    raise ValueError('Unknown type %r' % type(v))
ValueError: Unknown type <class 'NoneType'>

2024-04-24 17:15:00,532 ERROR scheduler Exception in Enqueue Events for Site erpnext14.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 94, in enqueue_events
    queued_jobs = get_jobs(site=site, key="job_type").get(site) or []
  File "apps/frappe/frappe/utils/background_jobs.py", line 298, in get_jobs
    jobs = q.jobs + get_running_jobs_in_queue(q)
  File "apps/frappe/frappe/utils/background_jobs.py", line 335, in get_running_jobs_in_queue
    workers = get_workers(queue)
  File "apps/frappe/frappe/utils/background_jobs.py", line 327, in get_workers
    return Worker.all(queue=queue)
  File "env/lib/python3.10/site-packages/rq/worker.py", line 257, in all
    workers = [
  File "env/lib/python3.10/site-packages/rq/worker.py", line 258, in <listcomp>
    cls.find_by_key(
  File "env/lib/python3.10/site-packages/rq/worker.py", line 685, in find_by_key
    worker.refresh()
  File "env/lib/python3.10/site-packages/rq/worker.py", line 1041, in refresh
    queues = as_text(queues)
  File "env/lib/python3.10/site-packages/rq/utils.py", line 58, in as_text
    raise ValueError('Unknown type %r' % type(v))
ValueError: Unknown type <class 'NoneType'>

2024-04-24 17:16:00,582 ERROR scheduler Exception in Enqueue Events for Site erpnext14.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 94, in enqueue_events
    queued_jobs = get_jobs(site=site, key="job_type").get(site) or []
  File "apps/frappe/frappe/utils/background_jobs.py", line 298, in get_jobs
    jobs = q.jobs + get_running_jobs_in_queue(q)
  File "apps/frappe/frappe/utils/background_jobs.py", line 335, in get_running_jobs_in_queue
    workers = get_workers(queue)
  File "apps/frappe/frappe/utils/background_jobs.py", line 327, in get_workers
    return Worker.all(queue=queue)
  File "env/lib/python3.10/site-packages/rq/worker.py", line 257, in all
    workers = [
  File "env/lib/python3.10/site-packages/rq/worker.py", line 258, in <listcomp>
    cls.find_by_key(
  File "env/lib/python3.10/site-packages/rq/worker.py", line 685, in find_by_key
    worker.refresh()
  File "env/lib/python3.10/site-packages/rq/worker.py", line 1041, in refresh
    queues = as_text(queues)
  File "env/lib/python3.10/site-packages/rq/utils.py", line 58, in as_text
    raise ValueError('Unknown type %r' % type(v))
ValueError: Unknown type <class 'NoneType'>

2024-04-24 17:17:00,625 ERROR scheduler Exception in Enqueue Events for Site erpnext14.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 94, in enqueue_events
    queued_jobs = get_jobs(site=site, key="job_type").get(site) or []
  File "apps/frappe/frappe/utils/background_jobs.py", line 298, in get_jobs
    jobs = q.jobs + get_running_jobs_in_queue(q)
  File "apps/frappe/frappe/utils/background_jobs.py", line 335, in get_running_jobs_in_queue
    workers = get_workers(queue)
  File "apps/frappe/frappe/utils/background_jobs.py", line 327, in get_workers
    return Worker.all(queue=queue)
  File "env/lib/python3.10/site-packages/rq/worker.py", line 257, in all
    workers = [
  File "env/lib/python3.10/site-packages/rq/worker.py", line 258, in <listcomp>
    cls.find_by_key(
  File "env/lib/python3.10/site-packages/rq/worker.py", line 685, in find_by_key
    worker.refresh()
  File "env/lib/python3.10/site-packages/rq/worker.py", line 1041, in refresh
    queues = as_text(queues)
  File "env/lib/python3.10/site-packages/rq/utils.py", line 58, in as_text
    raise ValueError('Unknown type %r' % type(v))
ValueError: Unknown type <class 'NoneType'>

2024-04-24 17:18:00,674 ERROR scheduler Exception in Enqueue Events for Site erpnext14.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 94, in enqueue_events
    queued_jobs = get_jobs(site=site, key="job_type").get(site) or []
  File "apps/frappe/frappe/utils/background_jobs.py", line 298, in get_jobs
    jobs = q.jobs + get_running_jobs_in_queue(q)
  File "apps/frappe/frappe/utils/background_jobs.py", line 335, in get_running_jobs_in_queue
    workers = get_workers(queue)
  File "apps/frappe/frappe/utils/background_jobs.py", line 327, in get_workers
    return Worker.all(queue=queue)
  File "env/lib/python3.10/site-packages/rq/worker.py", line 257, in all
    workers = [
  File "env/lib/python3.10/site-packages/rq/worker.py", line 258, in <listcomp>
    cls.find_by_key(
  File "env/lib/python3.10/site-packages/rq/worker.py", line 685, in find_by_key
    worker.refresh()
  File "env/lib/python3.10/site-packages/rq/worker.py", line 1041, in refresh
    queues = as_text(queues)
  File "env/lib/python3.10/site-packages/rq/utils.py", line 58, in as_text
    raise ValueError('Unknown type %r' % type(v))
ValueError: Unknown type <class 'NoneType'>

2024-04-24 17:19:00,730 ERROR scheduler Exception in Enqueue Events for Site erpnext14.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 94, in enqueue_events
    queued_jobs = get_jobs(site=site, key="job_type").get(site) or []
  File "apps/frappe/frappe/utils/background_jobs.py", line 298, in get_jobs
    jobs = q.jobs + get_running_jobs_in_queue(q)
  File "apps/frappe/frappe/utils/background_jobs.py", line 335, in get_running_jobs_in_queue
    workers = get_workers(queue)
  File "apps/frappe/frappe/utils/background_jobs.py", line 327, in get_workers
    return Worker.all(queue=queue)
  File "env/lib/python3.10/site-packages/rq/worker.py", line 257, in all
    workers = [
  File "env/lib/python3.10/site-packages/rq/worker.py", line 258, in <listcomp>
    cls.find_by_key(
  File "env/lib/python3.10/site-packages/rq/worker.py", line 685, in find_by_key
    worker.refresh()
  File "env/lib/python3.10/site-packages/rq/worker.py", line 1041, in refresh
    queues = as_text(queues)
  File "env/lib/python3.10/site-packages/rq/utils.py", line 58, in as_text
    raise ValueError('Unknown type %r' % type(v))
ValueError: Unknown type <class 'NoneType'>

2024-04-24 17:20:00,848 ERROR scheduler Exception in Enqueue Events for Site erpnext14.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 94, in enqueue_events
    queued_jobs = get_jobs(site=site, key="job_type").get(site) or []
  File "apps/frappe/frappe/utils/background_jobs.py", line 298, in get_jobs
    jobs = q.jobs + get_running_jobs_in_queue(q)
  File "apps/frappe/frappe/utils/background_jobs.py", line 335, in get_running_jobs_in_queue
    workers = get_workers(queue)
  File "apps/frappe/frappe/utils/background_jobs.py", line 327, in get_workers
    return Worker.all(queue=queue)
  File "env/lib/python3.10/site-packages/rq/worker.py", line 257, in all
    workers = [
  File "env/lib/python3.10/site-packages/rq/worker.py", line 258, in <listcomp>
    cls.find_by_key(
  File "env/lib/python3.10/site-packages/rq/worker.py", line 685, in find_by_key
    worker.refresh()
  File "env/lib/python3.10/site-packages/rq/worker.py", line 1041, in refresh
    queues = as_text(queues)
  File "env/lib/python3.10/site-packages/rq/utils.py", line 58, in as_text
    raise ValueError('Unknown type %r' % type(v))
ValueError: Unknown type <class 'NoneType'>

2024-04-24 17:21:00,914 ERROR scheduler Exception in Enqueue Events for Site erpnext14.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 94, in enqueue_events
    queued_jobs = get_jobs(site=site, key="job_type").get(site) or []
  File "apps/frappe/frappe/utils/background_jobs.py", line 298, in get_jobs
    jobs = q.jobs + get_running_jobs_in_queue(q)
  File "apps/frappe/frappe/utils/background_jobs.py", line 335, in get_running_jobs_in_queue
    workers = get_workers(queue)
  File "apps/frappe/frappe/utils/background_jobs.py", line 327, in get_workers
    return Worker.all(queue=queue)
  File "env/lib/python3.10/site-packages/rq/worker.py", line 257, in all
    workers = [
  File "env/lib/python3.10/site-packages/rq/worker.py", line 258, in <listcomp>
    cls.find_by_key(
  File "env/lib/python3.10/site-packages/rq/worker.py", line 685, in find_by_key
    worker.refresh()
  File "env/lib/python3.10/site-packages/rq/worker.py", line 1041, in refresh
    queues = as_text(queues)
  File "env/lib/python3.10/site-packages/rq/utils.py", line 58, in as_text
    raise ValueError('Unknown type %r' % type(v))
ValueError: Unknown type <class 'NoneType'>

2024-04-24 17:22:01,032 ERROR scheduler Exception in Enqueue Events for Site erpnext14.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 94, in enqueue_events
    queued_jobs = get_jobs(site=site, key="job_type").get(site) or []
  File "apps/frappe/frappe/utils/background_jobs.py", line 298, in get_jobs
    jobs = q.jobs + get_running_jobs_in_queue(q)
  File "apps/frappe/frappe/utils/background_jobs.py", line 335, in get_running_jobs_in_queue
    workers = get_workers(queue)
  File "apps/frappe/frappe/utils/background_jobs.py", line 327, in get_workers
    return Worker.all(queue=queue)
  File "env/lib/python3.10/site-packages/rq/worker.py", line 257, in all
    workers = [
  File "env/lib/python3.10/site-packages/rq/worker.py", line 258, in <listcomp>
    cls.find_by_key(
  File "env/lib/python3.10/site-packages/rq/worker.py", line 685, in find_by_key
    worker.refresh()
  File "env/lib/python3.10/site-packages/rq/worker.py", line 1041, in refresh
    queues = as_text(queues)
  File "env/lib/python3.10/site-packages/rq/utils.py", line 58, in as_text
    raise ValueError('Unknown type %r' % type(v))
ValueError: Unknown type <class 'NoneType'>

2024-04-24 17:23:01,151 ERROR scheduler Exception in Enqueue Events for Site erpnext14.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 94, in enqueue_events
    queued_jobs = get_jobs(site=site, key="job_type").get(site) or []
  File "apps/frappe/frappe/utils/background_jobs.py", line 298, in get_jobs
    jobs = q.jobs + get_running_jobs_in_queue(q)
  File "apps/frappe/frappe/utils/background_jobs.py", line 335, in get_running_jobs_in_queue
    workers = get_workers(queue)
  File "apps/frappe/frappe/utils/background_jobs.py", line 327, in get_workers
    return Worker.all(queue=queue)
  File "env/lib/python3.10/site-packages/rq/worker.py", line 257, in all
    workers = [
  File "env/lib/python3.10/site-packages/rq/worker.py", line 258, in <listcomp>
    cls.find_by_key(
  File "env/lib/python3.10/site-packages/rq/worker.py", line 685, in find_by_key
    worker.refresh()
  File "env/lib/python3.10/site-packages/rq/worker.py", line 1041, in refresh
    queues = as_text(queues)
  File "env/lib/python3.10/site-packages/rq/utils.py", line 58, in as_text
    raise ValueError('Unknown type %r' % type(v))
ValueError: Unknown type <class 'NoneType'>

2024-04-25 07:46:28,929 ERROR scheduler Exception in Enqueue Events for Site erpnext14.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 94, in enqueue_events
    queued_jobs = get_jobs(site=site, key="job_type").get(site) or []
  File "apps/frappe/frappe/utils/background_jobs.py", line 298, in get_jobs
    jobs = q.jobs + get_running_jobs_in_queue(q)
  File "apps/frappe/frappe/utils/background_jobs.py", line 335, in get_running_jobs_in_queue
    workers = get_workers(queue)
  File "apps/frappe/frappe/utils/background_jobs.py", line 327, in get_workers
    return Worker.all(queue=queue)
  File "env/lib/python3.10/site-packages/rq/worker.py", line 257, in all
    workers = [
  File "env/lib/python3.10/site-packages/rq/worker.py", line 258, in <listcomp>
    cls.find_by_key(
  File "env/lib/python3.10/site-packages/rq/worker.py", line 685, in find_by_key
    worker.refresh()
  File "env/lib/python3.10/site-packages/rq/worker.py", line 1041, in refresh
    queues = as_text(queues)
  File "env/lib/python3.10/site-packages/rq/utils.py", line 58, in as_text
    raise ValueError('Unknown type %r' % type(v))
ValueError: Unknown type <class 'NoneType'>

2024-04-25 07:47:29,302 ERROR scheduler Exception in Enqueue Events for Site erpnext14.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 94, in enqueue_events
    queued_jobs = get_jobs(site=site, key="job_type").get(site) or []
  File "apps/frappe/frappe/utils/background_jobs.py", line 298, in get_jobs
    jobs = q.jobs + get_running_jobs_in_queue(q)
  File "apps/frappe/frappe/utils/background_jobs.py", line 335, in get_running_jobs_in_queue
    workers = get_workers(queue)
  File "apps/frappe/frappe/utils/background_jobs.py", line 327, in get_workers
    return Worker.all(queue=queue)
  File "env/lib/python3.10/site-packages/rq/worker.py", line 257, in all
    workers = [
  File "env/lib/python3.10/site-packages/rq/worker.py", line 258, in <listcomp>
    cls.find_by_key(
  File "env/lib/python3.10/site-packages/rq/worker.py", line 685, in find_by_key
    worker.refresh()
  File "env/lib/python3.10/site-packages/rq/worker.py", line 1041, in refresh
    queues = as_text(queues)
  File "env/lib/python3.10/site-packages/rq/utils.py", line 58, in as_text
    raise ValueError('Unknown type %r' % type(v))
ValueError: Unknown type <class 'NoneType'>

2024-04-25 07:48:29,404 ERROR scheduler Exception in Enqueue Events for Site erpnext14.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 94, in enqueue_events
    queued_jobs = get_jobs(site=site, key="job_type").get(site) or []
  File "apps/frappe/frappe/utils/background_jobs.py", line 298, in get_jobs
    jobs = q.jobs + get_running_jobs_in_queue(q)
  File "apps/frappe/frappe/utils/background_jobs.py", line 335, in get_running_jobs_in_queue
    workers = get_workers(queue)
  File "apps/frappe/frappe/utils/background_jobs.py", line 327, in get_workers
    return Worker.all(queue=queue)
  File "env/lib/python3.10/site-packages/rq/worker.py", line 257, in all
    workers = [
  File "env/lib/python3.10/site-packages/rq/worker.py", line 258, in <listcomp>
    cls.find_by_key(
  File "env/lib/python3.10/site-packages/rq/worker.py", line 685, in find_by_key
    worker.refresh()
  File "env/lib/python3.10/site-packages/rq/worker.py", line 1041, in refresh
    queues = as_text(queues)
  File "env/lib/python3.10/site-packages/rq/utils.py", line 58, in as_text
    raise ValueError('Unknown type %r' % type(v))
ValueError: Unknown type <class 'NoneType'>

2024-04-25 07:49:29,485 ERROR scheduler Exception in Enqueue Events for Site erpnext14.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 94, in enqueue_events
    queued_jobs = get_jobs(site=site, key="job_type").get(site) or []
  File "apps/frappe/frappe/utils/background_jobs.py", line 298, in get_jobs
    jobs = q.jobs + get_running_jobs_in_queue(q)
  File "apps/frappe/frappe/utils/background_jobs.py", line 335, in get_running_jobs_in_queue
    workers = get_workers(queue)
  File "apps/frappe/frappe/utils/background_jobs.py", line 327, in get_workers
    return Worker.all(queue=queue)
  File "env/lib/python3.10/site-packages/rq/worker.py", line 257, in all
    workers = [
  File "env/lib/python3.10/site-packages/rq/worker.py", line 258, in <listcomp>
    cls.find_by_key(
  File "env/lib/python3.10/site-packages/rq/worker.py", line 685, in find_by_key
    worker.refresh()
  File "env/lib/python3.10/site-packages/rq/worker.py", line 1041, in refresh
    queues = as_text(queues)
  File "env/lib/python3.10/site-packages/rq/utils.py", line 58, in as_text
    raise ValueError('Unknown type %r' % type(v))
ValueError: Unknown type <class 'NoneType'>

2024-04-25 07:50:29,557 ERROR scheduler Exception in Enqueue Events for Site erpnext14.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 94, in enqueue_events
    queued_jobs = get_jobs(site=site, key="job_type").get(site) or []
  File "apps/frappe/frappe/utils/background_jobs.py", line 298, in get_jobs
    jobs = q.jobs + get_running_jobs_in_queue(q)
  File "apps/frappe/frappe/utils/background_jobs.py", line 335, in get_running_jobs_in_queue
    workers = get_workers(queue)
  File "apps/frappe/frappe/utils/background_jobs.py", line 327, in get_workers
    return Worker.all(queue=queue)
  File "env/lib/python3.10/site-packages/rq/worker.py", line 257, in all
    workers = [
  File "env/lib/python3.10/site-packages/rq/worker.py", line 258, in <listcomp>
    cls.find_by_key(
  File "env/lib/python3.10/site-packages/rq/worker.py", line 685, in find_by_key
    worker.refresh()
  File "env/lib/python3.10/site-packages/rq/worker.py", line 1041, in refresh
    queues = as_text(queues)
  File "env/lib/python3.10/site-packages/rq/utils.py", line 58, in as_text
    raise ValueError('Unknown type %r' % type(v))
ValueError: Unknown type <class 'NoneType'>

2024-04-25 07:51:29,633 ERROR scheduler Exception in Enqueue Events for Site erpnext14.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 94, in enqueue_events
    queued_jobs = get_jobs(site=site, key="job_type").get(site) or []
  File "apps/frappe/frappe/utils/background_jobs.py", line 298, in get_jobs
    jobs = q.jobs + get_running_jobs_in_queue(q)
  File "apps/frappe/frappe/utils/background_jobs.py", line 335, in get_running_jobs_in_queue
    workers = get_workers(queue)
  File "apps/frappe/frappe/utils/background_jobs.py", line 327, in get_workers
    return Worker.all(queue=queue)
  File "env/lib/python3.10/site-packages/rq/worker.py", line 257, in all
    workers = [
  File "env/lib/python3.10/site-packages/rq/worker.py", line 258, in <listcomp>
    cls.find_by_key(
  File "env/lib/python3.10/site-packages/rq/worker.py", line 685, in find_by_key
    worker.refresh()
  File "env/lib/python3.10/site-packages/rq/worker.py", line 1041, in refresh
    queues = as_text(queues)
  File "env/lib/python3.10/site-packages/rq/utils.py", line 58, in as_text
    raise ValueError('Unknown type %r' % type(v))
ValueError: Unknown type <class 'NoneType'>

2024-04-25 07:52:29,675 ERROR scheduler Exception in Enqueue Events for Site erpnext14.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 94, in enqueue_events
    queued_jobs = get_jobs(site=site, key="job_type").get(site) or []
  File "apps/frappe/frappe/utils/background_jobs.py", line 298, in get_jobs
    jobs = q.jobs + get_running_jobs_in_queue(q)
  File "apps/frappe/frappe/utils/background_jobs.py", line 335, in get_running_jobs_in_queue
    workers = get_workers(queue)
  File "apps/frappe/frappe/utils/background_jobs.py", line 327, in get_workers
    return Worker.all(queue=queue)
  File "env/lib/python3.10/site-packages/rq/worker.py", line 257, in all
    workers = [
  File "env/lib/python3.10/site-packages/rq/worker.py", line 258, in <listcomp>
    cls.find_by_key(
  File "env/lib/python3.10/site-packages/rq/worker.py", line 685, in find_by_key
    worker.refresh()
  File "env/lib/python3.10/site-packages/rq/worker.py", line 1041, in refresh
    queues = as_text(queues)
  File "env/lib/python3.10/site-packages/rq/utils.py", line 58, in as_text
    raise ValueError('Unknown type %r' % type(v))
ValueError: Unknown type <class 'NoneType'>

2024-04-25 07:53:29,810 ERROR scheduler Exception in Enqueue Events for Site erpnext14.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 94, in enqueue_events
    queued_jobs = get_jobs(site=site, key="job_type").get(site) or []
  File "apps/frappe/frappe/utils/background_jobs.py", line 298, in get_jobs
    jobs = q.jobs + get_running_jobs_in_queue(q)
  File "apps/frappe/frappe/utils/background_jobs.py", line 335, in get_running_jobs_in_queue
    workers = get_workers(queue)
  File "apps/frappe/frappe/utils/background_jobs.py", line 327, in get_workers
    return Worker.all(queue=queue)
  File "env/lib/python3.10/site-packages/rq/worker.py", line 257, in all
    workers = [
  File "env/lib/python3.10/site-packages/rq/worker.py", line 258, in <listcomp>
    cls.find_by_key(
  File "env/lib/python3.10/site-packages/rq/worker.py", line 685, in find_by_key
    worker.refresh()
  File "env/lib/python3.10/site-packages/rq/worker.py", line 1041, in refresh
    queues = as_text(queues)
  File "env/lib/python3.10/site-packages/rq/utils.py", line 58, in as_text
    raise ValueError('Unknown type %r' % type(v))
ValueError: Unknown type <class 'NoneType'>

2024-04-25 07:54:29,923 ERROR scheduler Exception in Enqueue Events for Site erpnext14.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 94, in enqueue_events
    queued_jobs = get_jobs(site=site, key="job_type").get(site) or []
  File "apps/frappe/frappe/utils/background_jobs.py", line 298, in get_jobs
    jobs = q.jobs + get_running_jobs_in_queue(q)
  File "apps/frappe/frappe/utils/background_jobs.py", line 335, in get_running_jobs_in_queue
    workers = get_workers(queue)
  File "apps/frappe/frappe/utils/background_jobs.py", line 327, in get_workers
    return Worker.all(queue=queue)
  File "env/lib/python3.10/site-packages/rq/worker.py", line 257, in all
    workers = [
  File "env/lib/python3.10/site-packages/rq/worker.py", line 258, in <listcomp>
    cls.find_by_key(
  File "env/lib/python3.10/site-packages/rq/worker.py", line 685, in find_by_key
    worker.refresh()
  File "env/lib/python3.10/site-packages/rq/worker.py", line 1041, in refresh
    queues = as_text(queues)
  File "env/lib/python3.10/site-packages/rq/utils.py", line 58, in as_text
    raise ValueError('Unknown type %r' % type(v))
ValueError: Unknown type <class 'NoneType'>

2024-04-25 07:55:30,072 ERROR scheduler Exception in Enqueue Events for Site erpnext14.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 94, in enqueue_events
    queued_jobs = get_jobs(site=site, key="job_type").get(site) or []
  File "apps/frappe/frappe/utils/background_jobs.py", line 298, in get_jobs
    jobs = q.jobs + get_running_jobs_in_queue(q)
  File "apps/frappe/frappe/utils/background_jobs.py", line 335, in get_running_jobs_in_queue
    workers = get_workers(queue)
  File "apps/frappe/frappe/utils/background_jobs.py", line 327, in get_workers
    return Worker.all(queue=queue)
  File "env/lib/python3.10/site-packages/rq/worker.py", line 257, in all
    workers = [
  File "env/lib/python3.10/site-packages/rq/worker.py", line 258, in <listcomp>
    cls.find_by_key(
  File "env/lib/python3.10/site-packages/rq/worker.py", line 685, in find_by_key
    worker.refresh()
  File "env/lib/python3.10/site-packages/rq/worker.py", line 1041, in refresh
    queues = as_text(queues)
  File "env/lib/python3.10/site-packages/rq/utils.py", line 58, in as_text
    raise ValueError('Unknown type %r' % type(v))
ValueError: Unknown type <class 'NoneType'>

2024-04-25 07:56:30,147 ERROR scheduler Exception in Enqueue Events for Site erpnext14.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 94, in enqueue_events
    queued_jobs = get_jobs(site=site, key="job_type").get(site) or []
  File "apps/frappe/frappe/utils/background_jobs.py", line 298, in get_jobs
    jobs = q.jobs + get_running_jobs_in_queue(q)
  File "apps/frappe/frappe/utils/background_jobs.py", line 335, in get_running_jobs_in_queue
    workers = get_workers(queue)
  File "apps/frappe/frappe/utils/background_jobs.py", line 327, in get_workers
    return Worker.all(queue=queue)
  File "env/lib/python3.10/site-packages/rq/worker.py", line 257, in all
    workers = [
  File "env/lib/python3.10/site-packages/rq/worker.py", line 258, in <listcomp>
    cls.find_by_key(
  File "env/lib/python3.10/site-packages/rq/worker.py", line 685, in find_by_key
    worker.refresh()
  File "env/lib/python3.10/site-packages/rq/worker.py", line 1041, in refresh
    queues = as_text(queues)
  File "env/lib/python3.10/site-packages/rq/utils.py", line 58, in as_text
    raise ValueError('Unknown type %r' % type(v))
ValueError: Unknown type <class 'NoneType'>

2024-04-25 07:57:30,268 ERROR scheduler Exception in Enqueue Events for Site erpnext14.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 94, in enqueue_events
    queued_jobs = get_jobs(site=site, key="job_type").get(site) or []
  File "apps/frappe/frappe/utils/background_jobs.py", line 298, in get_jobs
    jobs = q.jobs + get_running_jobs_in_queue(q)
  File "apps/frappe/frappe/utils/background_jobs.py", line 335, in get_running_jobs_in_queue
    workers = get_workers(queue)
  File "apps/frappe/frappe/utils/background_jobs.py", line 327, in get_workers
    return Worker.all(queue=queue)
  File "env/lib/python3.10/site-packages/rq/worker.py", line 257, in all
    workers = [
  File "env/lib/python3.10/site-packages/rq/worker.py", line 258, in <listcomp>
    cls.find_by_key(
  File "env/lib/python3.10/site-packages/rq/worker.py", line 685, in find_by_key
    worker.refresh()
  File "env/lib/python3.10/site-packages/rq/worker.py", line 1041, in refresh
    queues = as_text(queues)
  File "env/lib/python3.10/site-packages/rq/utils.py", line 58, in as_text
    raise ValueError('Unknown type %r' % type(v))
ValueError: Unknown type <class 'NoneType'>

2024-04-25 07:58:30,353 ERROR scheduler Exception in Enqueue Events for Site erpnext14.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 94, in enqueue_events
    queued_jobs = get_jobs(site=site, key="job_type").get(site) or []
  File "apps/frappe/frappe/utils/background_jobs.py", line 298, in get_jobs
    jobs = q.jobs + get_running_jobs_in_queue(q)
  File "apps/frappe/frappe/utils/background_jobs.py", line 335, in get_running_jobs_in_queue
    workers = get_workers(queue)
  File "apps/frappe/frappe/utils/background_jobs.py", line 327, in get_workers
    return Worker.all(queue=queue)
  File "env/lib/python3.10/site-packages/rq/worker.py", line 257, in all
    workers = [
  File "env/lib/python3.10/site-packages/rq/worker.py", line 258, in <listcomp>
    cls.find_by_key(
  File "env/lib/python3.10/site-packages/rq/worker.py", line 685, in find_by_key
    worker.refresh()
  File "env/lib/python3.10/site-packages/rq/worker.py", line 1041, in refresh
    queues = as_text(queues)
  File "env/lib/python3.10/site-packages/rq/utils.py", line 58, in as_text
    raise ValueError('Unknown type %r' % type(v))
ValueError: Unknown type <class 'NoneType'>

2024-04-25 07:59:30,375 ERROR scheduler Exception in Enqueue Events for Site erpnext14.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 94, in enqueue_events
    queued_jobs = get_jobs(site=site, key="job_type").get(site) or []
  File "apps/frappe/frappe/utils/background_jobs.py", line 298, in get_jobs
    jobs = q.jobs + get_running_jobs_in_queue(q)
  File "apps/frappe/frappe/utils/background_jobs.py", line 335, in get_running_jobs_in_queue
    workers = get_workers(queue)
  File "apps/frappe/frappe/utils/background_jobs.py", line 327, in get_workers
    return Worker.all(queue=queue)
  File "env/lib/python3.10/site-packages/rq/worker.py", line 257, in all
    workers = [
  File "env/lib/python3.10/site-packages/rq/worker.py", line 258, in <listcomp>
    cls.find_by_key(
  File "env/lib/python3.10/site-packages/rq/worker.py", line 685, in find_by_key
    worker.refresh()
  File "env/lib/python3.10/site-packages/rq/worker.py", line 1041, in refresh
    queues = as_text(queues)
  File "env/lib/python3.10/site-packages/rq/utils.py", line 58, in as_text
    raise ValueError('Unknown type %r' % type(v))
ValueError: Unknown type <class 'NoneType'>

2024-04-25 08:00:30,465 ERROR scheduler Exception in Enqueue Events for Site erpnext14.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 94, in enqueue_events
    queued_jobs = get_jobs(site=site, key="job_type").get(site) or []
  File "apps/frappe/frappe/utils/background_jobs.py", line 298, in get_jobs
    jobs = q.jobs + get_running_jobs_in_queue(q)
  File "apps/frappe/frappe/utils/background_jobs.py", line 335, in get_running_jobs_in_queue
    workers = get_workers(queue)
  File "apps/frappe/frappe/utils/background_jobs.py", line 327, in get_workers
    return Worker.all(queue=queue)
  File "env/lib/python3.10/site-packages/rq/worker.py", line 257, in all
    workers = [
  File "env/lib/python3.10/site-packages/rq/worker.py", line 258, in <listcomp>
    cls.find_by_key(
  File "env/lib/python3.10/site-packages/rq/worker.py", line 685, in find_by_key
    worker.refresh()
  File "env/lib/python3.10/site-packages/rq/worker.py", line 1041, in refresh
    queues = as_text(queues)
  File "env/lib/python3.10/site-packages/rq/utils.py", line 58, in as_text
    raise ValueError('Unknown type %r' % type(v))
ValueError: Unknown type <class 'NoneType'>

2024-04-25 08:01:30,589 ERROR scheduler Exception in Enqueue Events for Site erpnext14.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 94, in enqueue_events
    queued_jobs = get_jobs(site=site, key="job_type").get(site) or []
  File "apps/frappe/frappe/utils/background_jobs.py", line 298, in get_jobs
    jobs = q.jobs + get_running_jobs_in_queue(q)
  File "apps/frappe/frappe/utils/background_jobs.py", line 335, in get_running_jobs_in_queue
    workers = get_workers(queue)
  File "apps/frappe/frappe/utils/background_jobs.py", line 327, in get_workers
    return Worker.all(queue=queue)
  File "env/lib/python3.10/site-packages/rq/worker.py", line 257, in all
    workers = [
  File "env/lib/python3.10/site-packages/rq/worker.py", line 258, in <listcomp>
    cls.find_by_key(
  File "env/lib/python3.10/site-packages/rq/worker.py", line 685, in find_by_key
    worker.refresh()
  File "env/lib/python3.10/site-packages/rq/worker.py", line 1041, in refresh
    queues = as_text(queues)
  File "env/lib/python3.10/site-packages/rq/utils.py", line 58, in as_text
    raise ValueError('Unknown type %r' % type(v))
ValueError: Unknown type <class 'NoneType'>

2024-04-25 08:02:30,678 ERROR scheduler Exception in Enqueue Events for Site erpnext14.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 94, in enqueue_events
    queued_jobs = get_jobs(site=site, key="job_type").get(site) or []
  File "apps/frappe/frappe/utils/background_jobs.py", line 298, in get_jobs
    jobs = q.jobs + get_running_jobs_in_queue(q)
  File "apps/frappe/frappe/utils/background_jobs.py", line 335, in get_running_jobs_in_queue
    workers = get_workers(queue)
  File "apps/frappe/frappe/utils/background_jobs.py", line 327, in get_workers
    return Worker.all(queue=queue)
  File "env/lib/python3.10/site-packages/rq/worker.py", line 257, in all
    workers = [
  File "env/lib/python3.10/site-packages/rq/worker.py", line 258, in <listcomp>
    cls.find_by_key(
  File "env/lib/python3.10/site-packages/rq/worker.py", line 685, in find_by_key
    worker.refresh()
  File "env/lib/python3.10/site-packages/rq/worker.py", line 1041, in refresh
    queues = as_text(queues)
  File "env/lib/python3.10/site-packages/rq/utils.py", line 58, in as_text
    raise ValueError('Unknown type %r' % type(v))
ValueError: Unknown type <class 'NoneType'>

2024-04-25 08:03:30,740 ERROR scheduler Exception in Enqueue Events for Site erpnext14.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 94, in enqueue_events
    queued_jobs = get_jobs(site=site, key="job_type").get(site) or []
  File "apps/frappe/frappe/utils/background_jobs.py", line 298, in get_jobs
    jobs = q.jobs + get_running_jobs_in_queue(q)
  File "apps/frappe/frappe/utils/background_jobs.py", line 335, in get_running_jobs_in_queue
    workers = get_workers(queue)
  File "apps/frappe/frappe/utils/background_jobs.py", line 327, in get_workers
    return Worker.all(queue=queue)
  File "env/lib/python3.10/site-packages/rq/worker.py", line 257, in all
    workers = [
  File "env/lib/python3.10/site-packages/rq/worker.py", line 258, in <listcomp>
    cls.find_by_key(
  File "env/lib/python3.10/site-packages/rq/worker.py", line 685, in find_by_key
    worker.refresh()
  File "env/lib/python3.10/site-packages/rq/worker.py", line 1041, in refresh
    queues = as_text(queues)
  File "env/lib/python3.10/site-packages/rq/utils.py", line 58, in as_text
    raise ValueError('Unknown type %r' % type(v))
ValueError: Unknown type <class 'NoneType'>

2024-04-25 08:04:30,857 ERROR scheduler Exception in Enqueue Events for Site erpnext14.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 94, in enqueue_events
    queued_jobs = get_jobs(site=site, key="job_type").get(site) or []
  File "apps/frappe/frappe/utils/background_jobs.py", line 298, in get_jobs
    jobs = q.jobs + get_running_jobs_in_queue(q)
  File "apps/frappe/frappe/utils/background_jobs.py", line 335, in get_running_jobs_in_queue
    workers = get_workers(queue)
  File "apps/frappe/frappe/utils/background_jobs.py", line 327, in get_workers
    return Worker.all(queue=queue)
  File "env/lib/python3.10/site-packages/rq/worker.py", line 257, in all
    workers = [
  File "env/lib/python3.10/site-packages/rq/worker.py", line 258, in <listcomp>
    cls.find_by_key(
  File "env/lib/python3.10/site-packages/rq/worker.py", line 685, in find_by_key
    worker.refresh()
  File "env/lib/python3.10/site-packages/rq/worker.py", line 1041, in refresh
    queues = as_text(queues)
  File "env/lib/python3.10/site-packages/rq/utils.py", line 58, in as_text
    raise ValueError('Unknown type %r' % type(v))
ValueError: Unknown type <class 'NoneType'>

2024-04-25 08:05:30,977 ERROR scheduler Exception in Enqueue Events for Site erpnext14.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 94, in enqueue_events
    queued_jobs = get_jobs(site=site, key="job_type").get(site) or []
  File "apps/frappe/frappe/utils/background_jobs.py", line 298, in get_jobs
    jobs = q.jobs + get_running_jobs_in_queue(q)
  File "apps/frappe/frappe/utils/background_jobs.py", line 335, in get_running_jobs_in_queue
    workers = get_workers(queue)
  File "apps/frappe/frappe/utils/background_jobs.py", line 327, in get_workers
    return Worker.all(queue=queue)
  File "env/lib/python3.10/site-packages/rq/worker.py", line 257, in all
    workers = [
  File "env/lib/python3.10/site-packages/rq/worker.py", line 258, in <listcomp>
    cls.find_by_key(
  File "env/lib/python3.10/site-packages/rq/worker.py", line 685, in find_by_key
    worker.refresh()
  File "env/lib/python3.10/site-packages/rq/worker.py", line 1041, in refresh
    queues = as_text(queues)
  File "env/lib/python3.10/site-packages/rq/utils.py", line 58, in as_text
    raise ValueError('Unknown type %r' % type(v))
ValueError: Unknown type <class 'NoneType'>

2024-04-25 08:06:31,070 ERROR scheduler Exception in Enqueue Events for Site erpnext14.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 94, in enqueue_events
    queued_jobs = get_jobs(site=site, key="job_type").get(site) or []
  File "apps/frappe/frappe/utils/background_jobs.py", line 298, in get_jobs
    jobs = q.jobs + get_running_jobs_in_queue(q)
  File "apps/frappe/frappe/utils/background_jobs.py", line 335, in get_running_jobs_in_queue
    workers = get_workers(queue)
  File "apps/frappe/frappe/utils/background_jobs.py", line 327, in get_workers
    return Worker.all(queue=queue)
  File "env/lib/python3.10/site-packages/rq/worker.py", line 257, in all
    workers = [
  File "env/lib/python3.10/site-packages/rq/worker.py", line 258, in <listcomp>
    cls.find_by_key(
  File "env/lib/python3.10/site-packages/rq/worker.py", line 685, in find_by_key
    worker.refresh()
  File "env/lib/python3.10/site-packages/rq/worker.py", line 1041, in refresh
    queues = as_text(queues)
  File "env/lib/python3.10/site-packages/rq/utils.py", line 58, in as_text
    raise ValueError('Unknown type %r' % type(v))
ValueError: Unknown type <class 'NoneType'>

2024-04-25 08:07:31,184 ERROR scheduler Exception in Enqueue Events for Site erpnext14.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 94, in enqueue_events
    queued_jobs = get_jobs(site=site, key="job_type").get(site) or []
  File "apps/frappe/frappe/utils/background_jobs.py", line 298, in get_jobs
    jobs = q.jobs + get_running_jobs_in_queue(q)
  File "apps/frappe/frappe/utils/background_jobs.py", line 335, in get_running_jobs_in_queue
    workers = get_workers(queue)
  File "apps/frappe/frappe/utils/background_jobs.py", line 327, in get_workers
    return Worker.all(queue=queue)
  File "env/lib/python3.10/site-packages/rq/worker.py", line 257, in all
    workers = [
  File "env/lib/python3.10/site-packages/rq/worker.py", line 258, in <listcomp>
    cls.find_by_key(
  File "env/lib/python3.10/site-packages/rq/worker.py", line 685, in find_by_key
    worker.refresh()
  File "env/lib/python3.10/site-packages/rq/worker.py", line 1041, in refresh
    queues = as_text(queues)
  File "env/lib/python3.10/site-packages/rq/utils.py", line 58, in as_text
    raise ValueError('Unknown type %r' % type(v))
ValueError: Unknown type <class 'NoneType'>

2024-04-25 08:08:31,297 ERROR scheduler Exception in Enqueue Events for Site erpnext14.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 94, in enqueue_events
    queued_jobs = get_jobs(site=site, key="job_type").get(site) or []
  File "apps/frappe/frappe/utils/background_jobs.py", line 298, in get_jobs
    jobs = q.jobs + get_running_jobs_in_queue(q)
  File "apps/frappe/frappe/utils/background_jobs.py", line 335, in get_running_jobs_in_queue
    workers = get_workers(queue)
  File "apps/frappe/frappe/utils/background_jobs.py", line 327, in get_workers
    return Worker.all(queue=queue)
  File "env/lib/python3.10/site-packages/rq/worker.py", line 257, in all
    workers = [
  File "env/lib/python3.10/site-packages/rq/worker.py", line 258, in <listcomp>
    cls.find_by_key(
  File "env/lib/python3.10/site-packages/rq/worker.py", line 685, in find_by_key
    worker.refresh()
  File "env/lib/python3.10/site-packages/rq/worker.py", line 1041, in refresh
    queues = as_text(queues)
  File "env/lib/python3.10/site-packages/rq/utils.py", line 58, in as_text
    raise ValueError('Unknown type %r' % type(v))
ValueError: Unknown type <class 'NoneType'>

2024-04-25 08:09:31,327 ERROR scheduler Exception in Enqueue Events for Site erpnext14.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 94, in enqueue_events
    queued_jobs = get_jobs(site=site, key="job_type").get(site) or []
  File "apps/frappe/frappe/utils/background_jobs.py", line 298, in get_jobs
    jobs = q.jobs + get_running_jobs_in_queue(q)
  File "apps/frappe/frappe/utils/background_jobs.py", line 335, in get_running_jobs_in_queue
    workers = get_workers(queue)
  File "apps/frappe/frappe/utils/background_jobs.py", line 327, in get_workers
    return Worker.all(queue=queue)
  File "env/lib/python3.10/site-packages/rq/worker.py", line 257, in all
    workers = [
  File "env/lib/python3.10/site-packages/rq/worker.py", line 258, in <listcomp>
    cls.find_by_key(
  File "env/lib/python3.10/site-packages/rq/worker.py", line 685, in find_by_key
    worker.refresh()
  File "env/lib/python3.10/site-packages/rq/worker.py", line 1041, in refresh
    queues = as_text(queues)
  File "env/lib/python3.10/site-packages/rq/utils.py", line 58, in as_text
    raise ValueError('Unknown type %r' % type(v))
ValueError: Unknown type <class 'NoneType'>

2024-04-25 08:10:31,417 ERROR scheduler Exception in Enqueue Events for Site erpnext14.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 94, in enqueue_events
    queued_jobs = get_jobs(site=site, key="job_type").get(site) or []
  File "apps/frappe/frappe/utils/background_jobs.py", line 298, in get_jobs
    jobs = q.jobs + get_running_jobs_in_queue(q)
  File "apps/frappe/frappe/utils/background_jobs.py", line 335, in get_running_jobs_in_queue
    workers = get_workers(queue)
  File "apps/frappe/frappe/utils/background_jobs.py", line 327, in get_workers
    return Worker.all(queue=queue)
  File "env/lib/python3.10/site-packages/rq/worker.py", line 257, in all
    workers = [
  File "env/lib/python3.10/site-packages/rq/worker.py", line 258, in <listcomp>
    cls.find_by_key(
  File "env/lib/python3.10/site-packages/rq/worker.py", line 685, in find_by_key
    worker.refresh()
  File "env/lib/python3.10/site-packages/rq/worker.py", line 1041, in refresh
    queues = as_text(queues)
  File "env/lib/python3.10/site-packages/rq/utils.py", line 58, in as_text
    raise ValueError('Unknown type %r' % type(v))
ValueError: Unknown type <class 'NoneType'>

2024-04-25 08:11:31,455 ERROR scheduler Exception in Enqueue Events for Site erpnext14.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 94, in enqueue_events
    queued_jobs = get_jobs(site=site, key="job_type").get(site) or []
  File "apps/frappe/frappe/utils/background_jobs.py", line 298, in get_jobs
    jobs = q.jobs + get_running_jobs_in_queue(q)
  File "apps/frappe/frappe/utils/background_jobs.py", line 335, in get_running_jobs_in_queue
    workers = get_workers(queue)
  File "apps/frappe/frappe/utils/background_jobs.py", line 327, in get_workers
    return Worker.all(queue=queue)
  File "env/lib/python3.10/site-packages/rq/worker.py", line 257, in all
    workers = [
  File "env/lib/python3.10/site-packages/rq/worker.py", line 258, in <listcomp>
    cls.find_by_key(
  File "env/lib/python3.10/site-packages/rq/worker.py", line 685, in find_by_key
    worker.refresh()
  File "env/lib/python3.10/site-packages/rq/worker.py", line 1041, in refresh
    queues = as_text(queues)
  File "env/lib/python3.10/site-packages/rq/utils.py", line 58, in as_text
    raise ValueError('Unknown type %r' % type(v))
ValueError: Unknown type <class 'NoneType'>

2024-04-25 08:12:31,567 ERROR scheduler Exception in Enqueue Events for Site erpnext14.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 94, in enqueue_events
    queued_jobs = get_jobs(site=site, key="job_type").get(site) or []
  File "apps/frappe/frappe/utils/background_jobs.py", line 298, in get_jobs
    jobs = q.jobs + get_running_jobs_in_queue(q)
  File "apps/frappe/frappe/utils/background_jobs.py", line 335, in get_running_jobs_in_queue
    workers = get_workers(queue)
  File "apps/frappe/frappe/utils/background_jobs.py", line 327, in get_workers
    return Worker.all(queue=queue)
  File "env/lib/python3.10/site-packages/rq/worker.py", line 257, in all
    workers = [
  File "env/lib/python3.10/site-packages/rq/worker.py", line 258, in <listcomp>
    cls.find_by_key(
  File "env/lib/python3.10/site-packages/rq/worker.py", line 685, in find_by_key
    worker.refresh()
  File "env/lib/python3.10/site-packages/rq/worker.py", line 1041, in refresh
    queues = as_text(queues)
  File "env/lib/python3.10/site-packages/rq/utils.py", line 58, in as_text
    raise ValueError('Unknown type %r' % type(v))
ValueError: Unknown type <class 'NoneType'>

2024-04-25 08:13:31,685 ERROR scheduler Exception in Enqueue Events for Site erpnext14.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 94, in enqueue_events
    queued_jobs = get_jobs(site=site, key="job_type").get(site) or []
  File "apps/frappe/frappe/utils/background_jobs.py", line 298, in get_jobs
    jobs = q.jobs + get_running_jobs_in_queue(q)
  File "apps/frappe/frappe/utils/background_jobs.py", line 335, in get_running_jobs_in_queue
    workers = get_workers(queue)
  File "apps/frappe/frappe/utils/background_jobs.py", line 327, in get_workers
    return Worker.all(queue=queue)
  File "env/lib/python3.10/site-packages/rq/worker.py", line 257, in all
    workers = [
  File "env/lib/python3.10/site-packages/rq/worker.py", line 258, in <listcomp>
    cls.find_by_key(
  File "env/lib/python3.10/site-packages/rq/worker.py", line 685, in find_by_key
    worker.refresh()
  File "env/lib/python3.10/site-packages/rq/worker.py", line 1041, in refresh
    queues = as_text(queues)
  File "env/lib/python3.10/site-packages/rq/utils.py", line 58, in as_text
    raise ValueError('Unknown type %r' % type(v))
ValueError: Unknown type <class 'NoneType'>

2024-04-25 08:14:31,726 ERROR scheduler Exception in Enqueue Events for Site erpnext14.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 94, in enqueue_events
    queued_jobs = get_jobs(site=site, key="job_type").get(site) or []
  File "apps/frappe/frappe/utils/background_jobs.py", line 298, in get_jobs
    jobs = q.jobs + get_running_jobs_in_queue(q)
  File "apps/frappe/frappe/utils/background_jobs.py", line 335, in get_running_jobs_in_queue
    workers = get_workers(queue)
  File "apps/frappe/frappe/utils/background_jobs.py", line 327, in get_workers
    return Worker.all(queue=queue)
  File "env/lib/python3.10/site-packages/rq/worker.py", line 257, in all
    workers = [
  File "env/lib/python3.10/site-packages/rq/worker.py", line 258, in <listcomp>
    cls.find_by_key(
  File "env/lib/python3.10/site-packages/rq/worker.py", line 685, in find_by_key
    worker.refresh()
  File "env/lib/python3.10/site-packages/rq/worker.py", line 1041, in refresh
    queues = as_text(queues)
  File "env/lib/python3.10/site-packages/rq/utils.py", line 58, in as_text
    raise ValueError('Unknown type %r' % type(v))
ValueError: Unknown type <class 'NoneType'>

2024-04-25 08:15:31,856 ERROR scheduler Exception in Enqueue Events for Site erpnext14.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 94, in enqueue_events
    queued_jobs = get_jobs(site=site, key="job_type").get(site) or []
  File "apps/frappe/frappe/utils/background_jobs.py", line 298, in get_jobs
    jobs = q.jobs + get_running_jobs_in_queue(q)
  File "apps/frappe/frappe/utils/background_jobs.py", line 335, in get_running_jobs_in_queue
    workers = get_workers(queue)
  File "apps/frappe/frappe/utils/background_jobs.py", line 327, in get_workers
    return Worker.all(queue=queue)
  File "env/lib/python3.10/site-packages/rq/worker.py", line 257, in all
    workers = [
  File "env/lib/python3.10/site-packages/rq/worker.py", line 258, in <listcomp>
    cls.find_by_key(
  File "env/lib/python3.10/site-packages/rq/worker.py", line 685, in find_by_key
    worker.refresh()
  File "env/lib/python3.10/site-packages/rq/worker.py", line 1041, in refresh
    queues = as_text(queues)
  File "env/lib/python3.10/site-packages/rq/utils.py", line 58, in as_text
    raise ValueError('Unknown type %r' % type(v))
ValueError: Unknown type <class 'NoneType'>

2024-04-25 08:16:31,973 ERROR scheduler Exception in Enqueue Events for Site erpnext14.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 94, in enqueue_events
    queued_jobs = get_jobs(site=site, key="job_type").get(site) or []
  File "apps/frappe/frappe/utils/background_jobs.py", line 298, in get_jobs
    jobs = q.jobs + get_running_jobs_in_queue(q)
  File "apps/frappe/frappe/utils/background_jobs.py", line 335, in get_running_jobs_in_queue
    workers = get_workers(queue)
  File "apps/frappe/frappe/utils/background_jobs.py", line 327, in get_workers
    return Worker.all(queue=queue)
  File "env/lib/python3.10/site-packages/rq/worker.py", line 257, in all
    workers = [
  File "env/lib/python3.10/site-packages/rq/worker.py", line 258, in <listcomp>
    cls.find_by_key(
  File "env/lib/python3.10/site-packages/rq/worker.py", line 685, in find_by_key
    worker.refresh()
  File "env/lib/python3.10/site-packages/rq/worker.py", line 1041, in refresh
    queues = as_text(queues)
  File "env/lib/python3.10/site-packages/rq/utils.py", line 58, in as_text
    raise ValueError('Unknown type %r' % type(v))
ValueError: Unknown type <class 'NoneType'>

2024-04-25 08:17:32,089 ERROR scheduler Exception in Enqueue Events for Site erpnext14.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 94, in enqueue_events
    queued_jobs = get_jobs(site=site, key="job_type").get(site) or []
  File "apps/frappe/frappe/utils/background_jobs.py", line 298, in get_jobs
    jobs = q.jobs + get_running_jobs_in_queue(q)
  File "apps/frappe/frappe/utils/background_jobs.py", line 335, in get_running_jobs_in_queue
    workers = get_workers(queue)
  File "apps/frappe/frappe/utils/background_jobs.py", line 327, in get_workers
    return Worker.all(queue=queue)
  File "env/lib/python3.10/site-packages/rq/worker.py", line 257, in all
    workers = [
  File "env/lib/python3.10/site-packages/rq/worker.py", line 258, in <listcomp>
    cls.find_by_key(
  File "env/lib/python3.10/site-packages/rq/worker.py", line 685, in find_by_key
    worker.refresh()
  File "env/lib/python3.10/site-packages/rq/worker.py", line 1041, in refresh
    queues = as_text(queues)
  File "env/lib/python3.10/site-packages/rq/utils.py", line 58, in as_text
    raise ValueError('Unknown type %r' % type(v))
ValueError: Unknown type <class 'NoneType'>

2024-04-25 08:18:32,147 ERROR scheduler Exception in Enqueue Events for Site erpnext14.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 94, in enqueue_events
    queued_jobs = get_jobs(site=site, key="job_type").get(site) or []
  File "apps/frappe/frappe/utils/background_jobs.py", line 298, in get_jobs
    jobs = q.jobs + get_running_jobs_in_queue(q)
  File "apps/frappe/frappe/utils/background_jobs.py", line 335, in get_running_jobs_in_queue
    workers = get_workers(queue)
  File "apps/frappe/frappe/utils/background_jobs.py", line 327, in get_workers
    return Worker.all(queue=queue)
  File "env/lib/python3.10/site-packages/rq/worker.py", line 257, in all
    workers = [
  File "env/lib/python3.10/site-packages/rq/worker.py", line 258, in <listcomp>
    cls.find_by_key(
  File "env/lib/python3.10/site-packages/rq/worker.py", line 685, in find_by_key
    worker.refresh()
  File "env/lib/python3.10/site-packages/rq/worker.py", line 1041, in refresh
    queues = as_text(queues)
  File "env/lib/python3.10/site-packages/rq/utils.py", line 58, in as_text
    raise ValueError('Unknown type %r' % type(v))
ValueError: Unknown type <class 'NoneType'>

2024-04-25 08:19:32,270 ERROR scheduler Exception in Enqueue Events for Site erpnext14.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 94, in enqueue_events
    queued_jobs = get_jobs(site=site, key="job_type").get(site) or []
  File "apps/frappe/frappe/utils/background_jobs.py", line 298, in get_jobs
    jobs = q.jobs + get_running_jobs_in_queue(q)
  File "apps/frappe/frappe/utils/background_jobs.py", line 335, in get_running_jobs_in_queue
    workers = get_workers(queue)
  File "apps/frappe/frappe/utils/background_jobs.py", line 327, in get_workers
    return Worker.all(queue=queue)
  File "env/lib/python3.10/site-packages/rq/worker.py", line 257, in all
    workers = [
  File "env/lib/python3.10/site-packages/rq/worker.py", line 258, in <listcomp>
    cls.find_by_key(
  File "env/lib/python3.10/site-packages/rq/worker.py", line 685, in find_by_key
    worker.refresh()
  File "env/lib/python3.10/site-packages/rq/worker.py", line 1041, in refresh
    queues = as_text(queues)
  File "env/lib/python3.10/site-packages/rq/utils.py", line 58, in as_text
    raise ValueError('Unknown type %r' % type(v))
ValueError: Unknown type <class 'NoneType'>

2024-04-25 08:20:32,306 ERROR scheduler Exception in Enqueue Events for Site erpnext14.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 94, in enqueue_events
    queued_jobs = get_jobs(site=site, key="job_type").get(site) or []
  File "apps/frappe/frappe/utils/background_jobs.py", line 298, in get_jobs
    jobs = q.jobs + get_running_jobs_in_queue(q)
  File "apps/frappe/frappe/utils/background_jobs.py", line 335, in get_running_jobs_in_queue
    workers = get_workers(queue)
  File "apps/frappe/frappe/utils/background_jobs.py", line 327, in get_workers
    return Worker.all(queue=queue)
  File "env/lib/python3.10/site-packages/rq/worker.py", line 257, in all
    workers = [
  File "env/lib/python3.10/site-packages/rq/worker.py", line 258, in <listcomp>
    cls.find_by_key(
  File "env/lib/python3.10/site-packages/rq/worker.py", line 685, in find_by_key
    worker.refresh()
  File "env/lib/python3.10/site-packages/rq/worker.py", line 1041, in refresh
    queues = as_text(queues)
  File "env/lib/python3.10/site-packages/rq/utils.py", line 58, in as_text
    raise ValueError('Unknown type %r' % type(v))
ValueError: Unknown type <class 'NoneType'>

2024-04-25 08:21:32,325 ERROR scheduler Exception in Enqueue Events for Site erpnext14.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 94, in enqueue_events
    queued_jobs = get_jobs(site=site, key="job_type").get(site) or []
  File "apps/frappe/frappe/utils/background_jobs.py", line 298, in get_jobs
    jobs = q.jobs + get_running_jobs_in_queue(q)
  File "apps/frappe/frappe/utils/background_jobs.py", line 335, in get_running_jobs_in_queue
    workers = get_workers(queue)
  File "apps/frappe/frappe/utils/background_jobs.py", line 327, in get_workers
    return Worker.all(queue=queue)
  File "env/lib/python3.10/site-packages/rq/worker.py", line 257, in all
    workers = [
  File "env/lib/python3.10/site-packages/rq/worker.py", line 258, in <listcomp>
    cls.find_by_key(
  File "env/lib/python3.10/site-packages/rq/worker.py", line 685, in find_by_key
    worker.refresh()
  File "env/lib/python3.10/site-packages/rq/worker.py", line 1041, in refresh
    queues = as_text(queues)
  File "env/lib/python3.10/site-packages/rq/utils.py", line 58, in as_text
    raise ValueError('Unknown type %r' % type(v))
ValueError: Unknown type <class 'NoneType'>

2024-04-25 08:22:32,381 ERROR scheduler Exception in Enqueue Events for Site erpnext14.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 94, in enqueue_events
    queued_jobs = get_jobs(site=site, key="job_type").get(site) or []
  File "apps/frappe/frappe/utils/background_jobs.py", line 298, in get_jobs
    jobs = q.jobs + get_running_jobs_in_queue(q)
  File "apps/frappe/frappe/utils/background_jobs.py", line 335, in get_running_jobs_in_queue
    workers = get_workers(queue)
  File "apps/frappe/frappe/utils/background_jobs.py", line 327, in get_workers
    return Worker.all(queue=queue)
  File "env/lib/python3.10/site-packages/rq/worker.py", line 257, in all
    workers = [
  File "env/lib/python3.10/site-packages/rq/worker.py", line 258, in <listcomp>
    cls.find_by_key(
  File "env/lib/python3.10/site-packages/rq/worker.py", line 685, in find_by_key
    worker.refresh()
  File "env/lib/python3.10/site-packages/rq/worker.py", line 1041, in refresh
    queues = as_text(queues)
  File "env/lib/python3.10/site-packages/rq/utils.py", line 58, in as_text
    raise ValueError('Unknown type %r' % type(v))
ValueError: Unknown type <class 'NoneType'>

